index.html

```diff
@@ -22,7 +22,16 @@
     <div id='page'>
 <h1>Welcome to the Tamarin Manual</h1>
 <p class="halfbreak">
-The manual is available as a <a href='tex/tamarin-manual.pdf'>PDF</a> or as <a href='book/001_introduction.html'>HTML</a> for online browsing.
+The manual is available as a
+<a href='master/tex/tamarin-manual.pdf'>PDF</a>
+or as 
+<a href='master/book/001_introduction.html'>HTML</a>
+for online browsing. 
+(For developers, see here for the 
+<a href='develop/tex/tamarin-manual.pdf'>PDF</a>
+and
+<a href='develop/book/001_introduction.html'>HTML</a>
+that is in sync with the <tt>develop</tt> branch.)
```

Comment:
For many people the semantics of "the develop branch" might not be clear. What is this, how do I get it, why should I (not) use it?

Reply:
<img width="758" alt="grafik" src="https://user-images.githubusercontent.com/2065833/219395440-6440a2a9-945f-4361-846a-ddab12aed1e8.png">

That's what I had in mind. ([618f1ae](https://github.com/tamarin-prover/manual/pull/101/commits/618f1aed6246d779f616ec2ad1775b51561e68b8))

Reply:
Looks good!